### PR TITLE
RIP-529, /official_sections: manually added instructor can see existing sections

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira-secure.berkeley.edu/browse/RIP-529

### CODE REVIEW

File: ripley/lib/canvas_site_utils.py [hunk 1]
```diff
@@ -215,30 +210,32 @@ def get_teaching_terms(current_user=None, section_ids=None, sections=None, term_
...
-        teaching_sections = sections
+    term_ids = [t.to_sis_term_id() for t in terms]
+    teaching_sections = data_loch.get_instructing_sections(instructor_uid, term_ids) if instructor_uid else []
+    if sections:
+        for section in sections:
+            section_id = section['section_id']
+            term_id = section['term_id']
+            match = next((s for s in teaching_sections if s['term_id'] == term_id and s['section_id'] == section_id), None)
+            if not match:
+                teaching_sections.append(section)
```
Comment (johncrossman) [inline ripley/lib/canvas_site_utils.py:221]: The fix is above. We include `sections` (see incoming param) not represented in `teaching_sections`.

File: src/views/ManageOfficialSections.vue [hunk 2]
```diff
@@ -66,6 +66,7 @@
               Cancel<span class="sr-only"> edit sections</span>
             </v-btn>
           </div>
+          XXX
```
Comment (lyttam) [inline src/views/ManageOfficialSections.vue]: @johncrossman is this a typo?

Comment (johncrossman) [reply src/views/ManageOfficialSections.vue]: LOL! Yes it is.

**Comment by lyttam** [comment]
Thank you for the fix!

**Pull request merged**